pkg/cmd/util/clientcmd/factory.go

```diff
@@ -95,41 +106,20 @@ func NewFactory(clientConfig kclientcmd.ClientConfig) *Factory {
 		return describe.NewHumanReadablePrinter(noHeaders), nil
 	}
 
-	w.DefaultNamespace = func() (string, error) {
-		return w.OpenShiftClientConfig.Namespace()
```

Comment:
The namespace should come from the default factory, no need to override it.

----- Original Message -----
> > @@ -95,41 +106,20 @@ func NewFactory(clientConfig kclientcmd.ClientConfig)
> > *Factory {
> >  		return describe.NewHumanReadablePrinter(noHeaders), nil
> >  	}
> >  
> > -	w.DefaultNamespace = func() (string, error) {
> > -		return w.OpenShiftClientConfig.Namespace()
> 
> why was this wrong?
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/openshift/origin/pull/1436/files#r27127249